components/brave_wallet/browser/json_rpc_response_parser.cc

```diff
@@ -34,16 +35,12 @@ bool ParseSingleStringResult(const std::string& json, std::string* result) {
   return true;
 }
 
-}  // namespace
-
-namespace brave_wallet {
-
 void ParseErrorResult(const std::string& json,
```

Comment:
Maybe call this `ParseErrorResultCommon`

Reply:
You might even be able to just have the one original function and then make it take a template for the enum type name.

Reply:
Used a template in 90f4f9463a3e4b53e2e912dcb337a0fcfdf7fd94.